cli.js

```diff
@@ -96,6 +97,10 @@ function run(file) {
 		args.push('--fail-fast');
 	}
 
+	if (cli.flags.serial) {
+		args.push('--serial');
+	}
```

Comment:
Ooh, this will not look good... I propose to postpone this, until we have more options, what do you say?